netpyne/network/conn.py

```diff
@@ -1,16 +1,10 @@
-
 """
 network/conn.py 
 
 Network class methods to create connections 
 
 Contributors: salvadordura@gmail.com
 """
-from __future__ import print_function
```

Comment:
These all seem to be boilerplate to ensure Python 2 <--> 3 compatibility.  I suggest we leave them in unless there is a reproducible problem with them...

Reply:
i'm going to now try with new installs -- current netpyne, current ubuntu 20.04, and wheel-installed NEURON

Reply:
Sounds good. Let us know what happens.